bindings/python/libmata/nfa/nfa.pyx

```diff
@@ -1,5 +1,15 @@
-cimport libmata as mata
+cimport libmata.nfa.nfa as mata
```

Comment:
Should we not import `Nfa` "namespace" as something like `nfa`, `mata_nfa` or similar? It seems strange to import only NFAs as `mata`. Similarly to what we do with alphabets, importing them as `alph`. And of course, specific imports with `from ...` can then follow.

Reply:
Resolved below.